specification/templates/CONTRIBUTING.md

```diff
@@ -75,6 +75,6 @@ any 'help wanted' issues is a great place to start.
 See the [LICENSE](LICENSE) file for our repository's licensing. We will ask you to
 confirm the licensing of your contribution.
```

Comment:
Could we remove `We will ask you to confirm the licensing of your contribution.` if one already has to sign the CLA?